README.md

```diff
@@ -83,6 +83,48 @@ The following packages are suggested:
 [phpDox]: http://phpdox.de
 [Sami]: https://github.com/FriendsOfPHP/sami
 
+## Usage
+
+This is a simple metapackage, but still can be used in different ways.
+
+#### Global install
+
+This is by far the easiest and most convenient method. Simply having all the tools available system wide.
+
+The following script will install a system wide Composer for you, including the QA tools.
+
+```bash
+bash <(curl -S https://raw.githubusercontent.com/DealerDirect/php-qa-tools/master/bin/install.sh)
```

Comment:
Why not use:
```
wget -O install.sh https://raw.githubusercontent.com/DealerDirect/php-qa-tools/master/bin/install.sh
bash install.sh
```

Yes you will split it up but you will let the user decide if he wants to see the file with:
```
cat install.sh
```

You will give the power to the user to check that file for security reasons.